express/src/routes/create-account-or-sign-in.ts

```diff
@@ -15,12 +15,12 @@ import {
     submitMobileVerificationCode,
     updatePassword
 } from "../controllers/create-account";
-import {emailValidator} from "../middleware/emailValidator";
-import {mobileOtpValidator} from "../middleware/mobileOtpValidator";
-import notOnCommonPasswordListValidator from "../middleware/notOnCommonPasswordListValidator";
-import {passwordValidator} from "../middleware/passwordValidator";
-import {emailOtpValidator} from "../middleware/emailOtpValidator";
-import validateMobileNumber from "../middleware/mobileValidator";
+import {emailValidator} from "../validators/emailValidator";
```

Comment:
So that really, is a middleware that does validation using code in lib/validators.

The whole thing confuses me a bit which is why I've lent on you in the draft but that thing is more closely related to middleware than the utility code.  I'm sure at some point we said something about validators not being in middleware and maybe I got my wires crossed at some point so I moved the middleware that does validation (as opposed to the stuff that say checks authorisation) into its own folder.

Reply:
I'll have another fresh look tomorrow.